src/main/java/org/cbioportal/web/parameter/VirtualStudyData.java

```diff
@@ -24,6 +24,8 @@ public class VirtualStudyData implements Serializable {
     private String typeOfCancerId;
     private String pmid;
 
+    private Boolean dynamic;
```

Comment:
Is it possible to be NULL? is that why you are choosing not to use the primitive type boolean?

Reply:
@haynescd VS created before this change will have this field set to null. I am actually not sure what'd happen if this field would be primitive. I'd expect serialiser to complain